Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -464,7 +465,7 @@ await useCase.Execute(organisationId)
                 return operation;
             });
 
-        app.MapPost("/{organisationId}/persons/person-invite",
```

Comment:
Please can this be reverted back to what it was before. It will break the front end by changing this without updating the front end at the same time.

Reply:
It has been reverted back by @rmohammed-goaco for now but will update endpoints and corresponding front end in my next PR